esmvaltool/diag_scripts/iht_toa/poisson_solver.py

```diff
@@ -0,0 +1,285 @@
+"""(C) Crown Copyright 2023, the Met Office.
```

Comment:
I received a response from our legal team and "The short answer is yes, you need to include the Met Office copyright notice in your modification". I will plan to add one to the radiation budget recipe Jon and I contributed! ;)

However, I would recommend adding the copyright as a comment, rather than in the module docstring.

Reply:
Fixed in 53e7b08a8fe8c9b22547c242a71e60420fd9eda8